validation/manifest_validation_errors.go

```diff
@@ -0,0 +1,16 @@
+package validation
+
+import (
+	"errors"
+)
+
+var ErrInvalidManifest = errors.New("invalid manifest")
+var ErrGroupContainsNoServices = errors.New("The deployment group contains 0 services")
```

Comment:
Not entirely sure why, but Go community does prefer all lowercase error messages.

Reply:
yeah, i think it relates to parsing them

Reply:
Alright I'll change this.